common/lib/xmodule/xmodule/templates/problem/latex_problem.yaml

```diff
@@ -87,189 +87,173 @@ metadata:
-          <textline size="40" correct_answer="3" label="Integer #1"/><br/>
-          <textline size="40" correct_answer="7" label="Integer #2"/>
-        </customresponse>
-        <solution>
-          <div class="detailed-solution">
-            <p>Explanation</p>
-            <p>Any set of integers on the line \(y = 10 - x\) satisfy these constraints.</p>
-          </div>
-        </solution>
-      </question>
-
-      <br/>
-
-      <question>
-        <p>Enter two integers that sum to 20.</p>
-        <customresponse cfn="test_add" expect="20">
-          <textline size="40" correct_answer="11" label="Integer #1"/><br/>
-          <textline size="40" correct_answer="9" label="Integer #2"/>
-        </customresponse>
-        <solution>
-          <div class="detailed-solution">
-            <p>Explanation</p>
-            <p>Any set of integers on the line \(y = 20 - x\) satisfy these constraints.</p>
-            <p>To add an image to the solution, use an HTML "img" tag. Make sure to include alt text.</p>
-            <img src="/static/images/placeholder-image.png" width="400" alt="Description of image"/>
-          </div>
-        </solution>
-      </question>
-
-      <br/>
-
-      <question>
-        <p><strong>Example Image Mapped Input Problem</strong></p>
-        <p>
-          What country is home to the Great Pyramid of Giza as well as the cities
-          of Cairo and Memphis? Click the country on the map below.
-        </p>
-        <imageresponse>
-         <imageinput src="https://studio.edx.org/c4x/edX/DemoX/asset/Africa.png"
-         width="600" height="638" rectangle="(338,98)-(412,168)" alt="Map of
-         Africa"/>
-        </imageresponse>
-        <solution>
-          <div class="detailed-solution">
-            <p>Explanation</p>
-            <p>Egypt is home to not only the Pyramids, Cairo, and Memphis, but also
-            the Sphinx and the ancient Royal Library of Alexandria.</p>
-          </div>
-        </solution>
-      </question>
-
-      <br/>
-
-      <question>
-        <p><strong>Example Hidden Explanation</strong></p>
-        <p>You can provide additional information that only appears at certain times by including a "showhide" flag. </p>
-        <p>
-          <table class="wikitable collapsible collapsed">
-            <tbody>
-              <tr>
-                <th> More Information [<a href="javascript:$('#sh1').toggle()" id="sh1l">show</a>]</th>
-              </tr>
-              <tr id="sh1" style="display:none">
-                <td>
-                  <p>This is a hidden explanation. It can contain equations, such as [mathjaxinline]\alpha = \frac{2}{\sqrt {1+\gamma }}[/mathjaxinline].  </p>
-                  <p>This is additional text after the hidden explanation. </p>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </p>
-      </question>
-    </problem>
+      <?xml version="1.0"?>
+      <problem>
```

Comment:
@cpennington These attributes on problem element are valid only in case of course import/export. When course is exported, problem settings are set as attributes  on problem element and then during course import these attributes are removed from problem element. It looks like that someone mistakenly added these attributes here.